package/yast2-users.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-users
-Version:        4.1.12
+Version:        4.1.13
```

Comment:
Similar to https://github.com/yast/yast-pam/pull/13#pullrequestreview-227263255, very likely `4.2.0` should be here.